solutions/apigee/example/main.tf

```diff
@@ -0,0 +1,9 @@
+module "gce_instance" {
+  source = "github.com/CloudVLab/terraform-lab-foundation//solutions/apigee/dev"
+
+  # Pass values to the module
+  gcp_project_id = var.gcp_project_id
+  gcp_region     = var.gcp_region
+  gcp_zone       = var.gcp_zone
+  # depends_on     = []
```

Comment:
Should the metadata values not be passed in by the user of the module?

Reply:
Hi @rosera 
Please let us know what action to take here? Thanks!

Reply:
@mahakverma-tudip I believe the intention is to pass metadata values, therefore these should be defined in this variable?